charts/velero/templates/upgrade-crds/upgrade-crds.yaml

```diff
@@ -78,7 +52,7 @@ spec:
             - {{ .Values.upgradeCRDsJob.shellCmd | default "/tmp/sh" }}
```

Comment:
The velero image does not include a shell so you would have to change this command to something else. There's some more context on this issue here - https://github.com/vmware-tanzu/helm-charts/issues/571 - ultimately I'm not sure there is a good workaround currently to avoid the copying of a shell, without some changes in velero itself (such as https://github.com/vmware-tanzu/velero/pull/9132).

Reply:
I like your pull request, I hope it will be approve :)

Reply:
I have call directly velero

Reply:
So assuming that PR gets through it would just end up being `velero install --crds-only --apply` (with the current changes). No need for a kubectl image/container at all, the velero CLI will directly interact with the k8s API. It's hopefully a big simplification here.